reference_frame: map
time_to_wait: 1
goal_poses: [
  [0.4, -0.6928, -60], 
  [0, 0, 180], 
  [-0.8, 0, 180], 
  [0, 0, 60], 
  [0.4, 0.6928, 60], 
  [0, 0, 0], 
  [0.4, -0.6928, -60], 
  [0, 0, 180], 
  [-0.8, 0, 180], 
  [0, 0, 60], 
  [0.4, 0.6928, 60], 
  [0, 0, 0]
]

